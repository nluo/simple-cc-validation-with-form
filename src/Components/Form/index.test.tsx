import * as React from 'react';
import CCForm from './index';
const renderer = require('react-test-renderer');

it('renders correctly', () => {
  const tree = renderer
    .create(<CCForm/>) 
    .toJSON();
  expect(tree).toMatchSnapshot();
});