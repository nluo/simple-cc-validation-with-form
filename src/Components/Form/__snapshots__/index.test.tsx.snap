// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <form
    className="form"
    onSubmit={[Function]}
  >
    <div
      className="form-group form-header"
    >
      <h4>
         Payment Details 
      </h4>
    </div>
    <div
      className="form-group"
    >
      <label
        className="label"
      >
         
        Name On Card 
         
        <em>
          *
        </em>
      </label>
      <input
        className="form-control"
        name="cardHolderName"
        onBlur={undefined}
        onChange={[Function]}
        placeholder="First Name Last Name"
        required={true}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className="label"
      >
         
        Credit Card
         
        <em>
          *
        </em>
      </label>
      <input
        className="form-control"
        name="creditCardNumber"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder={undefined}
        required={true}
        type="text"
        value=""
      />
    </div>
    <div
      className="buttons-group"
    >
      <button
        className="btn btn-primary"
        type="submit"
      >
         Submit 
      </button>
    </div>
  </form>
  <div
    className="result"
  />
</div>
`;
